code/modules/virus2/effect.dm

```diff
@@ -70,344 +72,446 @@
 	chance_maxm = 0
 
 /datum/disease2/effect/gibbingtons
-	name = "Gibbingtons Syndrome"
+	name = "Gibbington's Syndrome"
 	stage = 4
 	badness = 3
-	activate(var/mob/living/carbon/mob,var/multiplier)
-		// Probabilities have been tweaked to kill in ~2-3 minutes, giving 5-10 messages.
-		// Probably needs more balancing, but it's better than LOL U GIBBED NOW, especially now that viruses can potentially have no signs up until Gibbingtons.
-		mob.adjustBruteLoss(10*multiplier)
-		if(istype(mob, /mob/living/carbon/human))
-			var/mob/living/carbon/human/H = mob
-			var/obj/item/organ/external/O = pick(H.organs)
-			if(prob(25))
-				mob << "<span class='warning'>Your [O.name] feels as if it might burst!</span>"
-			if(prob(10))
-				spawn(50)
-					if(O)
-						O.droplimb(0,DROPLIMB_BLUNT)
-		else
-			if(prob(75))
-				mob << "<span class='warning'>Your whole body feels like it might fall apart!</span>"
-			if(prob(10))
-				mob.adjustBruteLoss(25*multiplier)
+
+/datum/disease2/effect/gibbingtons/activate(var/mob/living/carbon/mob,var/multiplier)
+	// Probabilities have been tweaked to kill in ~2-3 minutes, giving 5-10 messages.
+	// Probably needs more balancing, but it's better than LOL U GIBBED NOW, especially now that viruses can potentially have no signs up until Gibbingtons.
+	mob.adjustBruteLoss(10*multiplier)
+	if(istype(mob, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = mob
+		var/obj/item/organ/external/O = pick(H.organs)
+		if(prob(25))
+			mob << "<span class='warning'>Your [O.name] feels as if it might burst!</span>"
+		if(prob(10))
+			spawn(50)
+				if(O)
+					O.droplimb(0,DROPLIMB_BLUNT)
+	else
+		if(prob(75))
+			mob << "<span class='warning'>Your whole body feels like it might fall apart!</span>"
+		if(prob(10))
+			mob.adjustBruteLoss(25*multiplier)
 
 /datum/disease2/effect/radian
 	name = "Radian's Syndrome"
 	stage = 4
 	maxm = 3
 	badness = 2
-	activate(var/mob/living/carbon/mob,var/multiplier)
-		mob.apply_effect(2*multiplier, IRRADIATE, check_protection = 0)
+
+/datum/disease2/effect/radian/activate(var/mob/living/carbon/mob,var/multiplier)
+	mob.apply_effect(2*multiplier, IRRADIATE, check_protection = 0)
 
 /datum/disease2/effect/deaf
```

Comment:
Not caused by you, but this seems to be the same datum path as the lesser deafness symptom.